src/components/Company/Offers/Manage/CompanyOffersManagementWidget.spec.js

```diff
@@ -6,12 +6,15 @@ import { renderWithStore } from "../../../../test-utils";
 import { MuiPickersUtilsProvider } from "@material-ui/pickers";
 import DateFnsUtils from "@date-io/date-fns";
 import * as companyOffersService from "../../../../services/companyOffersService";
-// import { fetchCompanyOffers } from "../../../../services/companyOffersService";
+import useSession from "../../../../hooks/useSession";
+
+jest.mock("../../../../hooks/useSession");
+jest.mock("../../../../services/companyOffersService");
 
 describe("App", () => {
     const MOCK_OFFERS = [
         {
-            id: "random uuid4",
+            _id: "random uuid4",
             title: "Guy in the background",
             ownerName: "Reddit",
             ownerLogo: "logo.com",
```

Comment:
The mock offers should have an `owner` property referring to the id of the company.